Dockerfile

```diff
@@ -1,18 +1,20 @@
-FROM python:3.12-alpine
+FROM python:3.12-rc-alpine
 
 WORKDIR /app
 
 # Define so the script knows not to download a new driver version, as
 # this Docker image already downloads a compatible chromedriver
 ENV AUTO_SOUTHWEST_CHECK_IN_DOCKER=1
 
-RUN apk add --update --no-cache chromium chromium-chromedriver
+RUN apk add --update --no-cache chromium chromium-chromedriver bash bash-completion nano
```

Comment:
Thanks for the fix! You can probably remove the extra dependencies added to the Dockerfile

Reply:
I don’t think we should install any dependencies not needed for production. It’s easy to modify the Docker image locally during development or install Bash once you already have a shell in the container.